libs/hdf-converters/package.json

```diff
@@ -26,7 +26,6 @@
   },
   "dependencies": {
     "@aws-sdk/client-config-service": "^3.95.0",
-    "@mitre/splunk-sdk-no-env": "^1.10.0",
```

Comment:
once this pr is merged we should have a plan of action to deprecate this library

Reply:
Did we publish this library to a public registry anywhere (like npm)?

Reply:
https://www.npmjs.com/package/@mitre/splunk-sdk-no-env